### PR TITLE
23 connect to server

### PR DESCRIPTION
**Pull request by furaiev**

### CODE REVIEW

File: src/components/TasksSummary/TasksSummary.js [hunk 1]
```diff
@@ -9,10 +10,14 @@ export class TasksSummary extends Component {
     userName: 'Alex'
```
Comment (iliatcymbal) [inline src/components/TasksSummary/TasksSummary.js]: _userName_ comes from server. You shouldn't hard code it, and other fields as well

File: src/services/rest.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { errObserver } from './observer';
+
+const BASE_URL = 'http://localhost:8082/';
```
Comment (iliatcymbal) [inline src/services/rest.js]: where did you get this url? It's wrong, use 8081, you can change it only for yourself locally

File: src/pages/Tasklist/Tasklist.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { Link } from 'react-router-dom';
+import { Tabs, Tab } from '../../components/Tabs';
+import { getTasks } from '../../services';
+import './Tasklist.scss';
+
+export class Tasklist extends Component {
+  state = {
+    days: ['Вс', 'Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб'],
```
Comment (iliatcymbal) [inline src/pages/Tasklist/Tasklist.js]: move it to separate file with constants

File: src/partials/Content/Content.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+export class Content extends Component {
+  state = {
+    tasks: false
+  }
+
+  render() {
+    return (
+      <div className="content">
+        <TasksSummary />
+        {this.state.tasks && <Redirect to="/tasks" />}
```
Comment (iliatcymbal) [inline src/partials/Content/Content.js]: What is this for?

File: src/services/rest.js (hunk 2 above)
Comment (furaiev) [reply src/services/rest.js]: :) ok, it was already used by other app

File: src/partials/Content/Content.js (hunk 4 above)
Comment (furaiev) [reply src/partials/Content/Content.js]: my bad

**Comment by furaiev** [comment]
@iliatcymbal updated

File: src/pages/Pages.js [hunk 5]
```diff
@@ -3,6 +3,6 @@ import { NonAuthorized } from '../Pages/NonAuthorized';
 
 export class Pages extends Component {
```
Comment (iliatcymbal) [inline src/pages/Pages.js]: no need class here

Comment (furaiev) [reply src/pages/Pages.js]: will be updated in the next PR

**Pull request merged**